tests/unit/expected_test_results/ingredients/fr-specific-ingredients.json

```diff
@@ -37,40 +37,60 @@
          "ingredients" : [
             {
                "id" : "en:fruit-pectin",
-               "percent_estimate" : 4.6875,
+               "percent_estimate" : 2.60416666666666,
                "percent_max" : 25,
                "percent_min" : 0,
                "text" : "pectines de fruits",
                "vegan" : "yes",
                "vegetarian" : "yes"
             },
             {
-               "id" : "fr:teneur-totale-en-sucres",
-               "percent_estimate" : 4.6875,
+               "id" : "fr:prepare-avec-32-5-de-legumes",
+               "percent_estimate" : 2.60416666666666,
                "percent_max" : 12.5,
                "percent_min" : 0,
-               "text" : "Teneur totale en sucres"
+               "text" : "Préparé avec 32.5% de légumes"
             }
          ],
-         "percent_estimate" : 9.375,
+         "percent_estimate" : 5.20833333333333,
          "percent_max" : 25,
          "percent_min" : 0,
          "text" : "gélifiant"
+      },
+      {
+         "id" : "fr:prepares-avec-25-2-de-tomates",
```

Comment:
Those cases used to be recognized before

Reply:
Just pushed correction to handle all those cases.

Reply:
Just saw that you opened a new PR 

Maybe you can merge manually changes of my last commit in the current PR: https://github.com/openfoodfacts/openfoodfacts-server/pull/9060/commits/cc54fd00855f162e29c56da7560a609792f5e7a1 into your new PR. 
And close this present PR.

	fr => "(?:élaboré|Elaboré|fabriqué|préparé|produit)(?:e)?(?:s)? (?:avec|à partir)", is missing prepare (without accents) in your PR which is in the test (PREPARE AVEC 30% DE TRUC INCONNU.)

this should work too, no?
(?:élaboré|Elaboré -> (?:(?:é|e)laboré

I will review your PR later this week.

Reply:
I added suggestion in your PR